src/utils/plotSpecs/generateFrequencySpec.js

```diff
@@ -63,7 +71,7 @@ const generateSpec = (config, plotData) => {
         range: 'height',
         nice: true,
         zero: true,
-        domain: { data: 'plotData', field: 'y1' },
+        domain: [0, 100],
```

Comment:
Does this line change not break the plot when you switch to count? The y axis should not be max 100 in this case.